packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx

```diff
@@ -15,12 +15,82 @@ import {
 } from '@lightdash/common';
 import React, { FC, useCallback, useMemo, useState } from 'react';
 import { useOrganisation } from '../../hooks/organisation/useOrganisation';
+import { useTracking } from '../../providers/TrackingProvider';
+import { EventName } from '../../types/Events';
 import { useVisualizationContext } from '../LightdashVisualization/VisualizationProvider';
-import { InputWrapper, Wrapper } from './ChartConfigPanel.styles';
+import {
+    InputWrapper,
+    MinMaxContainer,
+    MinMaxInput,
+    MinMaxWrapper,
+    Wrapper,
+} from './ChartConfigPanel.styles';
 import FieldLayoutOptions from './FieldLayoutOptions';
 import BasicSeriesConfiguration from './Series/BasicSeriesConfiguration';
 import GroupedSeriesConfiguration from './Series/GroupedSeriesConfiguration';
 
+interface MinMaxProps {
+    minVal: (index: number, value: string | number | undefined) => void;
+    maxVal: (index: number, value: string | number | undefined) => void;
+    index: number;
+    axis: string;
+}
+
+const AxisMinMax: FC<MinMaxProps> = ({ minVal, maxVal, index, axis }) => {
+    const [isAuto, setIsAuto] = useState<boolean>(true);
```

Comment:
Fyi: we have a useToggle hook for situations like this that might simplify the code slightly.

The isAuto is `true` every time we load the component. Should the value depend if there is a min/max value and remove the min/max values when user clicks back to be auto ?